galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/src/main/java/dev/galasa/docker/IDockerVolume.java

```diff
@@ -0,0 +1,30 @@
+package dev.galasa.docker;
+
+/** 
+ * A Galasa object to track, bind and provision Docker volumes with.
+ * 
+ * @author James Davies
+*/
+public interface IDockerVolume {
```

Comment:
How do you set the contents of the docker volume if it is a configuration volume